jsonschema/schema.go

```diff
@@ -10,27 +10,16 @@ import (
 
 // Instance represents a JSON Schema instance.
 type Instance struct {
-	Type       string                     `json:"type"`
-	Format     string                     `json:"format"`
-	Items      json.RawMessage            `json:"items,omitempty"`
-	Properties map[string]json.RawMessage `json:"properties,omitempty"`
-	Required   []string                   `json:"Required,omitempty"`
-}
-
-// schemaRef represents a JSON Schema reference.
-type schemaRef struct {
-	Ref string `json:"$ref"`
-}
-
-// schemaJSON represents the basic supported structure of a JSON Schema file
-type schemaJSON struct {
-	Instance
-
-	Schema      string      `json:"$schema"`
-	Description string      `json:"description,omitempty"`
-	AllOf       []schemaRef `json:"allOf,omitempty"`
-	OneOf       []schemaRef `json:"oneOf,omitempty"`
-	Required    []string    `json:"required,omitempty"`
+	Ref         string                     `json:"$ref,omitempty"`
```

Comment:
I like working toward more complete compliance with the full JSON schema spec. Clearly what was there before was a bit of a bare minimum, nice improvement.